tests/performance/test_coresubset.py

```diff
@@ -83,22 +82,21 @@ def test_greedy_body(self) -> None:
         post = []
         for i in range(self.num_samples_to_generate):
             # pylint: disable=protected-access
+            # fmt: off
```

Comment:
To prevent the keyword argument dictionary from being reformatted, which yields a line length greater than that permitted by pylint. Also, this is easier to read than the black formatting for this dict.